moveit_ros/moveit_servo/src/servo_calcs.cpp

```diff
@@ -555,11 +560,17 @@ bool ServoCalcs::cartesianServoCalcs(geometry_msgs::msg::TwistStamped& cmd,
     delta_theta_ = pseudo_inverse * delta_x;
   }
 
+  auto last_status = status_;
```

Comment:
const

Reply:
Removed the `auto` as well and replaced with the type